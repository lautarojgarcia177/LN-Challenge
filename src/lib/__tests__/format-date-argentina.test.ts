import formatDateArgentina from "../format-date-argentina";

describe("formatDateArgentina", () => {
  it("aplica formato a la fecha correctamente en EspaÃ±ol local de Argentina", () => {
    const date = new Date(2023, 9, 15);
    const formattedDate = formatDateArgentina(date);
    expect(formattedDate).toBe("15 de octubre de 2023");
  });

  it("aplica formato correctamente cuando la fecha tiene un solo digito de dia y de mes", () => {
    const date = new Date(2023, 8, 5);
    const formattedDate = formatDateArgentina(date);
    expect(formattedDate).toBe("5 de septiembre de 2023");
  });
  
  it("coincide con la snapshot", () => {
    const date = new Date(2023, 9, 15);
    const formattedDate = formatDateArgentina(date);
    expect(formattedDate).toMatchSnapshot();
  });
});
