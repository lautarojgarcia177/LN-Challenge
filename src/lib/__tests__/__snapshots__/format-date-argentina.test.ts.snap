// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`formatDateArgentina coincide con la snapshot 1`] = `"15 de octubre de 2023"`;
